### ISSUE TITLE
undefined value in connect function causes silent failure

### ISSUE THREAD

**Issue by ewiner** [heroku/react-refetch#129]
I had a bug where my `connect` function was returning something like `{"propName": undefined}`.  When that happened, React just silently failed to render the whole component tree.  There were no errors in the console, it just acted like I never called `ReactDOM.render()` on the top-level component.

Would it be possible to show an error in that circumstance?  I'm not sure if this is a problem with your library or with React, but it took a while to hunt down the issue.  I'm on React 15.1.0 and Chrome 51 if that helps.

**Comment by ryanbrainard** [comment]
Yes, I think that probably makes sense to emit a warning in that case. 

There was recent work (https://github.com/heroku/react-refetch/pull/128) on changing how `undefined` is handled for response transformations. I don't think that would have caused this, but might be worth try out the previous version (v1.0.0-beta.7) to see if that fixes the issues. What version were you using?

**Comment by ewiner** [comment]
I was on the latest, v1.0.0-beta.8.  I'll let you know how it goes if I try beta-7.
